packages/arb-token-bridge-ui/src/hooks/useArbTokenBridge.ts

```diff
@@ -443,6 +471,30 @@ export const useArbTokenBridge = (
         childChainId: Number(l2NetworkID)
       })
 
+      addDepositToCache({
+        sender: walletAddress,
+        destination: walletAddress,
+        status: 'pending',
+        txID: tx.hash,
+        assetName: symbol,
+        assetType: AssetType.ERC20,
+        l1NetworkID,
+        l2NetworkID,
```

Comment:
why do we need these when we already have `parentChainId` and `childChainId`?

Reply:
Legacy content that other legacy content depends on it, will remove with other PRs to clean things up.